packages/browser/package.json

```diff
@@ -40,6 +40,10 @@
     "./providers/playwright": {
       "types": "./providers/playwright.d.ts"
     },
+    "./locator": {
+      "types": "./dist/locators/index.d.ts",
+      "default": "./dist/locators/index.js"
+    },
```

Comment:
shouldn't be `locator`?

Can you add `typesVersions` for node10 below `"types"`?
```json
"typesVersions": {
    "*": {
      "client": [
        "./dist/client.d.ts"
      ],
      "locator": [
        "./dist/locators/index.d.ts"
      ]
    }
  },
```

Reply:
> Can you add `typesVersions` for node10 below `"types"`?

We don't support node 10

> shouldn't be `locator`?

What shouldn't be?

Reply:
It seems client subpackage is missing...

Reply:
client subpackage doesn't need a dts, it's not public

Reply:
`"moduleResolution": "node"` (areyourtypeswrong call it node10), add "locator" in "typesVersions"

Reply:
We don't support `moduleResolution: node` because Vite doesn't support it because Rollup doesn't support it (see `parseAst` exported from all)